src/java.base/share/classes/jdk/internal/module/SystemModuleFinders.java

```diff
@@ -253,6 +248,26 @@ private static ModuleFinder ofModuleInfos() {
         return new SystemModuleFinder(mrefs, nameToModule);
     }
 
+    private static Stream<ModuleInfo.Attributes> getModuleAttributes() {
```

Comment:
Can you rename this to allModuleAttributes and add a method description to match the other methods.